cpp/src/strings/regex/utilities.cuh

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <stdexcept>
```

Comment:
...and similarly. Move to the bottom in its own section, or with other STL includes.